packages/dashboard/src/project/projectListItem.tsx

```diff
@@ -33,11 +33,56 @@ type ProjectListItemProps = {
   reloadProjects: () => void;
 };
 
+/**
+ * Get tests information data from a given project.
+ * @param project the wanted project.
+ * @return the dataset of all tests status.
+ */
+export function useGetTestsFromProject(project: any) {
+  const projectId = project.projectId;
+  const search = '';
+  const [runsFeed] = useGetRunsFeed({
+    projectId,
+    search,
+  });
+  // We get the last run
+  const lastRun = runsFeed?.runs[0];
+  if (lastRun === undefined) return null;
+  const groups = lastRun?.progress?.groups;
+  if (!groups) return null;
+  return getRunTestsProgress(lastRun?.progress as RunProgress);
+}
+
+/**
+ * Get background color for the project component regarding the status of the test execution.
+ * @param project the wanted project.
+ * @return the wanted value (4 cases : empty project, failing project, flaky project, successful project)
+ */
+function useGetBackgroundColorRegardingTestStatus(
+  project: ArrayItemType<
+    Array<{
+      __typename?: 'Project';
+      projectId: string;
+      projectColor: string | null;
+    }>
+  >
+) {
+  const tests = useGetTestsFromProject(project);
+  // No test executed (empty project)
+  if (tests?.overall === undefined || tests?.overall === 0) return '#E7FFFD';
+  // At least 1 test failed
+  else if (tests?.failures) return '#FFC0C8';
+  // No fail but at least 1 test flaky
+  else if (tests?.flaky) return '#FADAF0';
+  // All tests worked
+  else return '#E0FEE7';
+}
+
 export function ProjectListItem({ project }: ProjectListItemProps) {
   // To fix a bug about project created directly from an execution instead of the dashboard
-  let defaultProjectColor;
-  if (project.projectColor === '') defaultProjectColor = '#3486E3';
-  else defaultProjectColor = project.projectColor ?? '#3486E3';
+  let defaultProjectNameColor;
+  if (project.projectColor === '') defaultProjectNameColor = '#3486E3';
+  else defaultProjectNameColor = project.projectColor ?? '#3486E3';
```

Comment:
@agoldis It's generally better to keep static colors within css rather than in code because it makes theming and dark mode code dependent. In this case you could use class toggles:
```
<Component className={classnames({
  "test-summary-failed": hasFailingTests,
  "test-summary-flaky": hasFlakyTests && !hasFailingTests,
  "test-summary-success": !hasFlakyTests && !hasFailingTests,
})} />
```